### PR TITLE
be able to set the schema registry compatibility via env var

### PR DESCRIPTION
**Pull request by galvo**
# About this change: What it does, why it matters

Require the ability to override the compatibility of the schema registry via env var.

(all contributors please complete this section, including maintainers)

### CODE REVIEW

File: container/start.sh [hunk 1]
```diff
@@ -37,7 +38,7 @@ start_karapace_registry(){
     "group_id": "${KARAPACE_REGISTRY_GROUP_ID:-$KARAPACE_REGISTRY_GROUP_ID_DEFAULT}",
     "master_eligibility": ${KARAPACE_REGISTRY_MASTER_ELIGIBITY:-$KARAPACE_REGISTRY_MASTER_ELIGIBITY_DEFAULT},
     "topic_name": "${KARAPACE_REGISTRY_TOPIC_NAME:-$KARAPACE_REGISTRY_TOPIC_NAME_DEFAULT}",
-    "compatibility": "FULL",
+    "compatibility": "${KARAPACE_REGISTRY_COMPATIBILITY:-${KARAPACE_REGISTRY_COMPATIBILITY_DEFAULT}",
```
Comment (tvainika) [inline container/start.sh:41]:
```suggestion
    "compatibility": "${KARAPACE_REGISTRY_COMPATIBILITY:-$KARAPACE_REGISTRY_COMPATIBILITY_DEFAULT}",
```

You have syntax error here, but other than that the contribution makes sense

**Review by tvainika — CHANGES_REQUESTED** [review]
Thank you for the contribution, this looks simple and useful for some. However there was small syntax error, which needs to fixed before merging

File: container/start.sh (hunk 1 above)
Comment (galvo) [reply container/start.sh]: Thanks, fixed now

File: container/docker-compose.yml [hunk 2]
```diff
@@ -75,7 +75,8 @@ services:
       KARAPACE_REGISTRY_MASTER_ELIGIBITY: "true"
       KARAPACE_REGISTRY_TOPIC_NAME: _schemas
       KARAPACE_REGISTRY_LOG_LEVEL: WARNING
-
+      KARAPACE_REGISTRY_COMPATIBILITY: FULL
+
```
Comment (tvainika) [inline container/docker-compose.yml:79]: Extra trailing whitespace here fails our automated linter still.

Comment (galvo) [reply container/docker-compose.yml]: resolved now

**Review by tvainika — APPROVED** [review]

**Comment by galvo** [comment]
Does the docker image get built automatically based on this merge?

**Comment by hackaugusto** [comment]
> Does the docker image get built automatically based on this merge?

Not yet. You can generate the image manually with:

```shell
./container/build_image.sh $(git rev-parse HEAD)  
```

**Comment by galvo** [comment]
Ya we pull the image from docker hub now so don't want to revert back to building it ourselves.

What is the process for getting whats on master built and pushed to docker hub?

**Comment by hackaugusto** [comment]
> What is the process for getting whats on master built and pushed to docker hub?

ATM we update the container image once we make a new release on aiven.io . This is done because we want the docker image features to match what we have in production.

**Pull request merged**